components/movementsensor/gpsutils/ntrip.go

```diff
@@ -212,18 +212,15 @@ func (n *NtripInfo) GetStreamFromMountPoint(
 		rc, err = n.client.GetStream(n.MountPoint)
 		if err == nil {
 			success = true
+			logger.Debug("Connected to stream")
+			n.stream = rc
```

Comment:
Should we set the debug message after setting `n.stream`?

Reply:
:shrug: Assigning the variable cannot fail, and I'd be surprised if writing a log statement could fail. So, I suspect the order doesn't matter. I'm happy to switch it.

Reply:
🤷‍♀️ I can go either way. Don't have a strong preference for it. Just thought it made more sense to say that connected after setting the stream